sql/parser/Makefile

```diff
@@ -0,0 +1,3 @@
+sql.go: sql.y
+	go tool yacc -o sql.go sql.y
+	@echo "$$(echo '// Code generated by go yacc.' | cat - sql.go)" > sql.go
```

Comment:
The outer `echo` looks unnecessary; is it just building up a gigantic command line? 

`awk 'BEGIN { print "// Code generated by go yacc." } { print }' < sql.go > sql.go`? `sed` can do it too but it's uglier.